public/v4/apps/bookstack.yml

```diff
@@ -0,0 +1,60 @@
+captainVersion: 4
+services:
+    $$cap_appname-mariadb:
+        image: linuxserver/mariadb:110.4.14mariabionic-ls77
+        environment:
+            MYSQL_ROOT_PASSWORD: $$cap_DB_PASS
+            MYSQL_DATABASE: bookstack
+            MYSQL_USER: bookstack
+            MYSQL_PASSWORD: $$cap_DB_PASS
+            PGID: 1000
+            PUID: 1000
+            TZ: $$cap_timezone
+        volumes:
+            - $$cap_appname-mariadb-config:/config
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        environment:
+            DB_HOST: srv-captain--$$cap_appname-mariadb
+            DB_USER: bookstack
+            DB_PASS: $$cap_DB_PASS
+            DB_DATABASE: bookstack
+            DB_PORT: 3306
+            PGID: 1000
+            PUID: 1000
+            TZ: $$cap_timezone
+            APP_URL: $$cap_appurl
+        image: linuxserver/bookstack:v0.29.3-ls99
```

Comment:
Generally it's a good idea to leave the version as a variable with a proper default value.

Reply:
Ah yes, I saw that in some of the other configs. With BookStack, there isn't really a major shift with the versions (to my knowledge) so I didn't think this was important. Should I still add the option for it? 🤔

Reply:
Yea I still think it's a good idea, otherwise the end user has no idea what version they are installing. Plus, when a newer version comes out, they can choose to change the newer tag.

Reply:
Added in 62a78bf